src/overlays/actors/ovl_En_Toto/z_en_toto.c

```diff
@@ -21,86 +60,703 @@ const ActorInit En_Toto_InitVars = {
+            func_800B8500(&this->actor, globalCtx, 9999.9f, 9999.9f, 0);
+        } else {
+            this->actor.flags &= 0xFFFEFFFF;
+            func_800B8614(&this->actor, globalCtx, 50.0f);
+            if (globalCtx->sceneNum == 0x12) {
+                if (player->unk14B == 3) {
+                    if (!Flags_GetSwitch(globalCtx, this->actor.home.rot.x)) {
+                        this->text = D_80BA5068;
+                    } else {
+                        this->text = D_80BA5074;
+                    }
+                } else {
+                    this->text = D_80BA5048;
+                }
+            } else if (ENTOTO_WEEK_EVENT_FLAGS) {
+                this->text = D_80BA502C;
+            } else if (!Flags_GetSwitch(globalCtx, this->actor.params & 0x7F)) {
+                this->text = D_80BA5034;
+            } else {
+                this->text = D_80BA5038;
+            }
+        }
+        this->actor.textId = this->text->textId;
+    }
+}
+
+void func_80BA3BFC(EnToto* this, GlobalContext* globalCtx) {
+    if (globalCtx->sceneNum == 0x12) {
+        SkelAnime_ChangeAnimTransitionStop(&this->skelAnime, &D_06000C80, -4.0f);
+        this->unk2B4 = 0;
+    } else {
+        if (this->text->unk0 == 4) {
+            func_80151BB4(globalCtx, 9);
+        }
+        SkelAnime_ChangeAnimTransitionRepeat(&this->skelAnime, &D_0600B3E0, -4.0f);
+    }
+}
+
+void func_80BA3C88(EnToto* this) {
+    Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 4, 4000, 800);
+}
+
+void func_80BA3CC4(EnToto* this, GlobalContext* globalCtx) {
+    func_80BA383C(this, globalCtx);
+    func_80BA3C88(this);
+    if (func_800B867C(&this->actor, globalCtx)) {
+        func_80BA36C0(this, globalCtx, this->text->unk1);
+    } else {
+        func_80BA4C44(this, globalCtx);
+    }
+}
+
+void func_80BA3D38(EnToto* this, GlobalContext* globalCtx) {
+    this->cutscene = this->actor.cutscene;
+    this->text = ENTOTO_WEEK_EVENT_FLAGS ? D_80BA50BC : D_80BA5088;
+    func_80BA4C0C(this, globalCtx);
+    globalCtx->actorCtx.unk5 |= 0x20;
+    this->unk260 = 0;
+}
+
+void func_80BA3DBC(EnToto* this, GlobalContext* globalCtx) {
+    ActorPlayer* player;
+
+    func_80BA383C(this, globalCtx);
+    if (this->unk2B7 == 0) {
+        if (!func_80BA4C44(this, globalCtx)) {
+            return;
+        }
+        if (this->text->unk1 != 0 && ENTOTO_WEEK_EVENT_FLAGS) {
+            this->unk2B7 = 1;
+            return;
+        }
+    } else {
+        player = PLAYER;
+        if (((player->stateFlags1 & 0x400) != 0) && (player->unkAE7 != 0)) {
+            func_80151BB4(globalCtx, 48);
+            func_80151BB4(globalCtx, 9);
+            func_80151BB4(globalCtx, 10);
+        } else {
+            return;
+        }
+    }
+    func_80BA36C0(this, globalCtx, 0);
+    ActorCutscene_Stop(this->cutscene);
+    globalCtx->actorCtx.unk5 &= 0xFFDF;
+}
+
+s32 func_80BA3EC0(EnToto* this, GlobalContext* globalCtx) {
+    return 1;
+}
```

Comment:
```suggestion
s32 func_80BA3EC0(EnToto* this, GlobalContext* globalCtx) {
    return true;
}
```
Looks like many of the following functions could return `true`/`false`

Reply:
On second thought, almost all  functions follow the `return 0` or `return 1` trend except `func_80BA4B24` inside `D_80BA512C` and `D_80BA5174`, so perhaps it might be better to leave this and my previous comment as `0` or `1` instead of `false`, `true`. It's tricky because following the trail on how these functions are used, the return's are only ever used as `true`/`false` if statements. Open to others thoughts.

Reply:
IMO, this seems like something to address in a documentation pass. Use 1/0 for true/false for now until we're confident.

Reply:
I agree with Kenix3. I would prefer more information/documentation before assigning these as booleans vs 0/1's.